vyper/compiler/output.py

```diff
@@ -206,7 +207,15 @@ def build_ir_runtime_dict_output(compiler_data: CompilerData) -> dict:
 
 
 def build_metadata_output(compiler_data: CompilerData) -> dict:
-    sigs = compiler_data.function_signatures
+    _ = compiler_data.function_signatures
+    module_t = compiler_data.annotated_vyper_module._metadata["type"]
+    sigs = dict[str, ContractFunctionT]()
+
+    for fn_t in module_t.exposed_functions:
+        assert isinstance(fn_t.ast_def, vy_ast.FunctionDef)
+        for inf_t in fn_t.reachable_internal_functions:
```

Comment:
what does `inf_t` stand for?

Reply:
internal function and the t cause its the type - ill rename it to something clearer